### ISSUE TITLE
Display diff -/+ so that the information is not illustrated in color only

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#115147]
Testing #114969

This is an accessibility feature:

![image](https://user-images.githubusercontent.com/2193314/105899300-5b538000-5fcf-11eb-9aa0-8c33571bc00c.png)

**Comment by rebornix** [comment]
Good point, it's not accessible.

**Comment by IanMatthewHuff** [comment]
![image](https://user-images.githubusercontent.com/812783/130695883-354fbfbb-10a2-453d-bf01-da912fd8d375.png)
Verified.
